PostHog/posthog
# feat: make links clickable in table

## Problem

Links in the table are not clickable. It would be more convenient if users could click on them. This was raised in an issue [here](https://github.com/PostHog/posthog/issues/20188).

## Changes
Links are now clickable in table entries and open in a new tab.
<img width="1440" alt="Screenshot 2024-04-30 at 20 24 12" src="https://github.com/PostHog/posthog/assets/27889631/3b62cb68-57d6-48d2-b92c-a3efa2c4ee26">


## Does this work well for both Cloud and self-hosted?

Yes

## How did you test this code?

- Run Storybook
- Open [Properties Table](http://localhost:6006/?path=/story/components-properties-table--basic)
- You can see the 3rd field with an URL that opens in a new tab
